### DISCUSSION TITLE
wednesday/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#114] [General]
# wednesday/

W tym poście podzielę się swoimi przemyśleniami na temat serialu telewizyjnego Wednesday na Netflixie. Serial, który bardzo mi się spodobał, różni się nieco od innych seriali z tego samego gatunku. Spektakl nie jest przeznaczony dla dzieci, ale warto go obejrzeć także dla dorosłych lubiących czarny humor i horror. Obsada jest znakomita, a scenariusz ostry. Nie mogę się doczekać kolejnego sezonu.

https://dariusz.wieckiewicz.org/wednesday/

**Comment by xpil** [comment]
Obejrzeliśmy - uchachaliśmy się po pachy - daje radę. Serial pełen niespodzianek. Mastroianni to to nie jest, ale nie zawsze musi być z klasą, żeby było zabawnie :)
